packages/SystemsThinkingWorkbench-Core.package/STWNode.class/instance/addMoveHandlerTo..st

```diff
@@ -0,0 +1,12 @@
+halos and balloon help
+addMoveHandlerTo: myHalo
+
+	|handle|
+	
+	handle := myHalo createHandleFor: #startDragTarget:  
+						to: myHalo
+						color: Color blueberry 
+						containing: (MenuIcons smallChangesIcon asMorph).
+	handle balloonText: 'Click me to move me around.'.
```

Comment:
"click me to do stuff to me" finde ich etwas unglücklich formuliert, da man ja auf den Halo clickt, aber die Node bearbeitet. Meinungen?